### PR TITLE
Makes the armblade handle more like the fireaxe with opening doors

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Armblade now pries open doors like the fireaxe rather than being an afterattack sort of thing
Deletes the old messages that you would get when arm-bladeing open a door. These were kinda cool but I really don't think they're necessary for a 10 second wait to open a door using the worlds shittiest jaws of life.  
Now works off a trait instead of hardcoded item stuff, feel free to give the clown the "infinity stamp" that opens up airlocks
Fixes a fuckup where I forgot to add a trait to the global trait list
This will also unintentionally fix a runtime

## Why It's Good For The Game
Two items doing near identical things outside of the time required to them isn't great. Internal consistency is important, and having these two things function in a much more similar way is good. 
Also lets admins do funny shit

## Testing
Compiled and ran, got ate by a lizard

## Changelog
:cl:
del: removed chat messages from the armblade opening up a door
tweak: Armblade functions more similarly to a fireaxe now, still can pry open unpowered doors and takes 10 seconds to open a powered one
/:cl:

### CODE REVIEW

File: code/_globalvars/traits.dm [hunk 1]
```diff
@@ -69,12 +69,14 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_AI_UNTRACKABLE" = TRAIT_AI_UNTRACKABLE,
 		"TRAIT_ELITE_CHALLENGER" = TRAIT_ELITE_CHALLENGER,
 		"TRAIT_SOAPY_MOUTH" = TRAIT_SOAPY_MOUTH,
-		"TRAIT_UNREVIVABLE" = TRAIT_UNREVIVABLE
+		"TRAIT_UNREVIVABLE" = TRAIT_UNREVIVABLE,
+		"TRAIT_CAN_BE_EATEN_BY_LIZARDS" = TRAIT_EDIBLE_BUG
```
Comment (farie82) [inline code/_globalvars/traits.dm:73]: Ehm... doesn't seem related to this change. Unless you employed lizards to eat the bugs you found while testing

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1161,20 +1161,31 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
...
+			if(!arePowerSystemsOn() && can_force_open_while_unpowered)
+				open(2)
+				return
+			prying_so_hard = TRUE //so you dont pry the door when you are already trying to pry it
+			var/result = do_after(user, time_to_open_airlock, target = src)
+			prying_so_hard = FALSE
+			if(result)
+				open(TRUE)
+				if(density && !open(TRUE))
+					to_chat(user, "<span class='warning'>Despite your attempts, [src] refuses to open.</span>")
```
Comment (farie82) [inline code/game/machinery/doors/airlock.dm:1188]: Can you make this into a proc? To keep the current proc a bit cleaner

File: code/_globalvars/traits.dm (hunk 1 above)
Comment (GDNgit) [reply code/_globalvars/traits.dm:73]: I forgot to do this in my previous pr 💀

**Comment by GDNgit** [comment]
> Can you please address farie's review?

I'll address it tonight I've just been lazy

File: code/game/machinery/doors/airlock.dm (hunk 2 above)
Comment (GDNgit) [reply code/game/machinery/doors/airlock.dm]: done

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -1220,6 +1208,32 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
...
+	var/time_to_open_airlock = 10 SECONDS
+	/// Can we open the airlock while unpowered? Wielded item's can't, but unwielded items can
+	var/can_force_open_while_unpowered = TRUE
+	if(istype(twohanded_item))
+		can_force_open_while_unpowered = FALSE
+		time_to_open_airlock = 5 SECONDS
+		if(!twohanded_item.wielded)
+			to_chat(user, "<span class='warning'>You need to be wielding [I] to do that!</span>")
+			return
+	if(density && !prying_so_hard)
```
Comment (farie82) [inline code/game/machinery/doors/airlock.dm:1224]: Can you make this into an early return instead?

Comment (GDNgit) [reply code/game/machinery/doors/airlock.dm]: done

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -1220,6 +1208,33 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
...
+		can_force_open_while_unpowered = FALSE
+		time_to_open_airlock = 5 SECONDS
+		if(!twohanded_item.wielded)
+			to_chat(user, "<span class='warning'>You need to be wielding [I] to do that!</span>")
+			return
+	if(!density || prying_so_hard)
+		return
+	playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, 1) //is it aliens or just the CE being a dick?
+	if(!arePowerSystemsOn() && can_force_open_while_unpowered)
+		open(2)
```
Comment (S34NW) [inline code/game/machinery/doors/airlock.dm:1228]:
```suggestion
		open(TRUE)
```
This is used as a bool elsewhere but as a int here, please correct one or the other

File: code/game/machinery/doors/airlock.dm [hunk 5]
```diff
@@ -1220,6 +1208,33 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
...
+		return
+	prying_so_hard = TRUE //so you dont pry the door when you are already trying to pry it
+	var/result = do_after(user, time_to_open_airlock, target = src)
+	prying_so_hard = FALSE
+	if(result)
+		open(TRUE)
```
Comment (S34NW) [inline code/game/machinery/doors/airlock.dm:1234]: here's the other open instance

**Review by S34NW — APPROVED** [review]

**Review by farie82 — APPROVED** [review]
